source/2019-11-01-ember-3-14-released.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Ember 3.14 Released (Octane Preview Cont.)
+author: Kenneth Larsen, Dan Gebhardt
+tags: Releases, 2019, 3, 3.14, Version 3.x
+responsive: true
+---
+
+Today the Ember project is releasing version 3.14 of Ember.js, Ember Data, and Ember CLI. This release kicks off the 3.15 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/builds/)
+- [The Ember Release Cycle](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html)
+- [The Ember Project](http://emberjs.com/blog/2015/06/16/ember-project-at-2-0.html)
+- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+
+---
+
+## The Octane Preview, Continued
+
+As described in yesterday's post, [Octane Release Update](/2019/10/31/octane-release-update.html), Octane will continue to be opt-in in Ember 3.14. By continuing the Octane preview period in 3.14, we are allowing more time for Octane to be polished before it becomes the default Ember experience.
+
+Although Octane is still opt-in in 3.14, all Octane features are stable public APIs with our normal SemVer guarantees. If you build an app using APIs in the Octane preview, it will continue to work in future versions.
+
+---
+
+With the exception of an updated release of `@glimmer/component`, the steps to try out the Octane preview remain the same in 3.14 as they were in 3.13:
+
+Add the following packages if they aren't already present at this version:
+
+```js
+npm install --save-dev @ember/edition-utils@^1.1.1
+npm install --save-dev @glimmer/component@^1.0.0-beta.1
+```
+
+Disable legacy behavior by setting the following feature flags:
+
+```bash
+ember feature:disable jquery-integration
```

Comment:
I thought disabling jQuery was no longer a requirement for opting into Octane? From https://blog.emberjs.com/2019/10/31/octane-release-update.html

> Because there is no technical reason to couple the removal of jQuery integration with Octane, we decided not to require Octane apps to disable jQuery integration. This requirement change is implemented as of the 3.14.0 release.

Reply:
@patocallaghan thanks - you're correct! I just carried this over from the 3.13 post.